_drafts/makefiles.md

```diff
@@ -0,0 +1,329 @@
+---
+layout: post
+title: Making Makefiles
+subtitle: Learn how to use Makefiles to automate any sort of task, including but
```

Comment:
Here's an example I just found in the Github terms:

> E2. Prices of all Services, including but not limited to monthly subscription plan fees to the Service, are subject to change upon 30 days notice from us. Such notice may be provided at any time by posting the changes to the GitHub Site (github.com) or the Service itself.

There's many more occurences. I think that is the idiom.